environments/sbox/sbox.tfvars

```diff
@@ -604,15 +604,15 @@ frontends = [
     shutter_app      = true
   },
   {
-    product          = "plum"
-    name             = "plumclassic"
-    custom_domain    = "frontdoor.sandbox.platform.hmcts.net"
-    dns_zone_name    = "frontdoor.sandbox.platform.hmcts.net"
-    backend_domain   = ["firewall-sbox-int-palo-sbox.uksouth.cloudapp.azure.com"]
-    certificate_name = "frontdoor-sandbox-platform-hmcts-net"
-    disabled_rules   = {}
-    shutter_app      = true
-    ssl_mode         = "AzureKeyVault"
+    product       = "plum"
+    name          = "plumclassic"
+    custom_domain = "hmcts-sbox.azurefd.net"
+    # dns_zone_name    = "frontdoor.sandbox.platform.hmcts.net"
```

Comment:
you need dns_zone_name for shutter app I think...

Reply:
just added now